Initialtask/src/Command/ProductsImportCommand.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace App\Command;
+
+use App\Entity\FileParser;
+use App\Entity\Product;
+use App\Repository\FileParserRepository;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+use Doctrine\ORM\EntityManagerInterface;
+
+
+class ProductsImportCommand extends Command
+{
+    protected static $defaultName = 'products:import';
+    protected static $defaultDescription = 'import products from *.csv file';
+
+    private $entityManager;
+
+    private $processedCount = 0;
+    private $skippedCount = 0;
+    private $successCount = 0;
+
+    public function __construct(EntityManagerInterface $em)
+    {
+        parent::__construct();
+
+        $this->EntityManager = $em;
+    }
+
+    protected function configure(): void
+    {
+        $this->addOption('testMode', null, InputOption::VALUE_NONE, 'is test mode enabled');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $io = new SymfonyStyle($input, $output);
+
+        $testMode = $input->getOption('testMode');  // if running:  products:import --testMode  == test mode;
+
+        $now = new \DateTimeImmutable('now');
+
+        $filePath = "%kernel.root_dir%/../public/stock.csv";    //CSV file for import
```

Comment:
guess, that path should be defined as parameter
https://symfony.com/doc/4.1/service_container/parameters.html